scvi/models/jvae.py

```diff
@@ -308,6 +308,9 @@ def decode(
 
         return px_scale, px_r, px_rate, px_dropout
 
+    def compute_global_kl_divergence(self):
```

Comment:
this does not look like it is called anywhere ?

Reply:
We do call it in `compute_elbo` in its current refactorized form